tests/draft2019-09/refRemote.json

```diff
@@ -163,5 +163,28 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "remote ref with ref to defs",
+        "schema": {
+            "$id": "http://localhost:1234/schema1.json",
+            "$ref": "ref-and-defs.json"
+        },
+        "tests": [
+            {
+                "description": "invalid",
+                "data": {
+                    "bar": 1
+                },
+                "valid": false
+            },
+            {
+                "description": "valid",
+                "data": {
+                    "bar": "a"
+                },
+                "valid": true
+            }
+        ]
     }
```

Comment:
I don't think this tests anything that the tests in "ref.json" don't. I'm not sure this is necessary.

Reply:
I added them mostly because I already added them to draft 7 and 6 (which can't have the embedded `$defs`). But I would expect resolution of an external document to be differently implemented than of an embedded one, which justifies testing it as well?

Reply:
> draft 7 and 6 (which can't have the embedded `$defs`)

Um, yes they can.

> I would expect resolution of an external document to be differently implemented than of an embedded one

The way I've implemented it, there is no difference, but it is possible that some implementations might treat them differently. So, I take your point.

Reply:
I tried those first (with embeded `definitions`), but they didn't work. I'll try it again, maybe this is also an implementation bug :)

Reply:
Got it working, thanks for noting this!